osgp/platform/osgp-domain-core/src/main/java/org/opensmartgridplatform/domain/core/valueobjects/RtuDevice.java

```diff
@@ -14,6 +14,11 @@
 
     private static final long serialVersionUID = -6133164707489276802L;
 
+    /**
+     * Device type indicator for Peak Shaving Device
+     */
+    public static final String PSD_TYPE = "PSD";
```

Comment:
For now this will probably sufficient, but I think we should have a look at how to handle all different kind of RTU device types in the future.